codegen-v2/src/codegen/templates/swift/enum.hbs

```diff
@@ -0,0 +1,27 @@
+// Copyright © 2017-2023 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+//
+// This is a GENERATED FILE, changes made here WILL BE LOST.
+//
```

Comment:
can we ensure the date of the copyright is always the current year?

Reply:
I can add that, sure.